src/Cms/App.php

```diff
@@ -825,12 +825,13 @@ public function languageCode(string $languageCode = null): ?string
     /**
      * Returns all available site languages
      *
+     * @param bool
```

Comment:
```suggestion
     * @param bool $clone
```

In which use-cases do we need the clone behavior? Maybe we could deprecate it in the future (in this case a simple TODO comment is enough for now).

Reply:
We used clone for App::languages() to protect the main languages collection from being accidentally overwritten, which would lead to many unwanted side-effects. But in all language actions, we actually need to overwrite it to make sure it's up to date. I.e. when a language gets deleted or updated. This is why I introduced a new clone prop. This way, I can get the uncloned instance of the collection and modify that. To be honest, I'm not 100% happy with this approach. But I didn't have an idea for a better solution. It's the typical dilemma that we have with our weird immutable objects everywhere.